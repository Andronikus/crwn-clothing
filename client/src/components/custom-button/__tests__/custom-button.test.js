import React from 'react';
import { render } from '@testing-library/react'
import TestRenderer from 'react-test-renderer';


import CustomButton from '../custom-button.component';

it('render properly the button component', () => {
    const buttonComponent = TestRenderer.create(<CustomButton>Sign In</CustomButton>).toJSON();
    expect(buttonComponent).toMatchSnapshot();
})
