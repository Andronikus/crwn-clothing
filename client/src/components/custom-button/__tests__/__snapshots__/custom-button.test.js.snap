// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render properly the button component 1`] = `
<button
  className="sc-AxjAm eOThmB"
>
  Sign In
</button>
`;
